pkgs/development/compilers/ghc/9.0.2.nix

```diff
@@ -182,6 +183,17 @@ stdenv.mkDerivation (rec {
 
   outputs = [ "out" "doc" ];
 
+  patches = [
+    # Add flag that fixes C++ exception handling; opt-in. Merged in 9.4 and 9.2.2.
+    # https://gitlab.haskell.org/ghc/ghc/-/merge_requests/7423
```

Comment:
I left a comment on the ghc MR w.r.t. documentation, given that it's a full rebuild to update it, can you take a look at that?

Reply:
I've addressed that comment in an extra upstream commit, which is not part of the `fetchpatch` below, so no rebuild is required.

This technically leaves the feature undocumented in our build, but I don't expect any significant number of people to use the built documentation in the store, let alone a user who needs to use the feature.